config.py

```diff
@@ -0,0 +1,58 @@
+import datetime
```

Comment:
We'll have two copies of this file now, so let's start thinking about how we can re-merge them in the future. Maybe these configurations could be accessible via admin and a request could be made to the server to grab those configuration settings?